lib/Runtime/Library/JavascriptString.cpp

```diff
@@ -2125,7 +2125,10 @@ namespace Js
 
         Assert(!(callInfo.Flags & CallFlags_New));
 
-        return ToLocaleCaseHelper(args[0], false, scriptContext);
+        JavascriptString * pThis = nullptr;
+        GetThisStringArgument(args, scriptContext, _u("String.prototype.toLocaleUpperCase"), &pThis);
```

Comment:
>toLocaleUpperCase [](start = 72, length = 17)

These labels are mixed up between the two functions.

Reply:
Good catch